lib/grandpa/message_handler_test.go

```diff
@@ -649,7 +649,7 @@ func TestMessageHandler_VerifyBlockJustification_WithEquivocatoryVotes(t *testin
 
 	round := uint64(1)
 	number := uint32(1)
-	precommits := buildTestJustification(t, 20, round, setID, kr, precommit)
+	precommits := buildTestJustification(t, 18, round, setID, kr, precommit)
```

Comment:
there are 9 authorities. If we vote count is 20, there would be few authorities with more than 2 votes. More than two equivocatory votes make the justification invalid.